src/stories/ButtonsShowroom.stories.tsx

```diff
@@ -29,6 +33,7 @@ const Template: StoryFn = () => {
       <GetStartedBtn>GetStartedBtn</GetStartedBtn>
       <ExploreBtn>ExploreBtn</ExploreBtn>
       <ZoomOutBtn>ZoomOutBtn</ZoomOutBtn>
+      <SelectNetworkButton isSelectedNetwork={false}>SelectNetworkButton</SelectNetworkButton>
```

Comment:
Hm, it doesn't look quite right, missing border even with isSelectedNetwork={true}
![image](https://github.com/oasisprotocol/explorer/assets/3758846/ac9244c9-21fd-43c5-8a16-5f2b3f24257f)